docker/web-and-data/Dockerfile

```diff
@@ -48,6 +48,7 @@ RUN wget https://github.com/cBioPortal/cbioportal-core/releases/download/v1.0.9/
 
 
 COPY --from=build /cbioportal/src/main/resources/db-scripts /cbioportal/db-scripts
+COPY --from=build /cbioportal/test/test_data/study_es_0 /cbioportal/test/study_es_0
```

Comment:
I'm not sure if the test data should go into the main app image. Kinda feels like it would be better in e.g. a pre populated mysql image

Just curious: why did u want to include it here?

Reply:
Ah nvm I see now. It was included originally before we moved core out. Ok that makes sense